### DISCUSSION TITLE
How can i close main window in android?

### DISCUSSION THREAD

**Discussion by alirezaimi** [beeware/toga#2006] [Q&A]
Hi 
How can i close main window ?
I read this https://toga.readthedocs.io/en/stable/reference/api/mainwindow.html but there is no close method in it, while in https://toga.readthedocs.io/en/stable/reference/api/window.html close method exist .

**Comment by mhsmith** [comment]
The `close` method isn't currently implemented on Android. See #1623 for details.

**Reply by alirezaimi** [reply]
Thanks, implemented in future release ?

**Reply by mhsmith** [reply]
We should be looking at this area later in 2023.

**Reply by alirezaimi** [reply]
thanks, can i work on some event like focus change to sleep app and stop it from running in background ?

**Reply by mhsmith** [reply]
We do receive those events from the sysem (onPause and onResume), but we don't currently expose them in the Toga API. I'm afraid I can't think of any workaround  for now, but I've created #2009 to track the idea.
